README.md

```diff
@@ -12,6 +12,10 @@ Current Build Status is: [![Build Status](https://secure.travis-ci.org/m-lab/m-l
 2. Run Jekyll server and pass in a blank baseurl to preview in development mode `jekyll serve --baseurl`.
 3. View the generated site by going to [http://localhost:4000/](http://localhost:4000/)
 
+### Pre-commit Hook
+
+Developers should also install the pre-commit hook that comes packaged with this repository that will alert about any trailing whitespaces to minimize git diff noise.  In order to install the pre-commit hook, run the following command to create a symbolic link: ```rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks/```.
```

Comment:
OK, that make sense.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/eabd7618abc1915efa0d31b50e1c9b29e56e6078 along with some other instances in the README where inline code had triple backticks.